ArraysHomework/ArraysHomework.cs

```diff
@@ -126,6 +141,8 @@ public void Lists()
 
             // Swap the last and first value of this list.
             List<string> swapMyValues = new List<string> { "first", "middle", "last" };
+            swapMyValues[0] = "last";
```

Comment:
How would you do this if you didn't know that the values were "first" and "last"? And if you didn't know the length of the List?